SPECS/xapi.spec

```diff
@@ -1368,6 +1375,14 @@ Coverage files from unit tests
 %{?_cov_results_package}
 
 %changelog
+* Thu Apr 03 2025 Guillaume Thouvenin <guillaume.thouvenin@vates.tech> - 24.39.1-1.3
+- Check that there are no changes during SR.scan
+- Improve the scan comparison logic
+- Add CAS style check for SR scan
+
+* Wed Apr 02 2025 Guillaume Thouvenin <guillaume.thouvenin@vates.tech> - 24.39.1-1.2
+- (this tag is not used)
+
```

Comment:
You can remove that. That's a dead release and we can jump directly from 1.1 to 1.3 in the changelog.